### ISSUE TITLE
Aragon Nest Proposal: Establishing a new Digital Body Corporate Legal Personality in the EU without Incorporation per #48, Aragon Nest proposal: DAOs and the interaction with the physical world

### ISSUE THREAD

**Issue by jtiannelli** [aragon/nest#59]
Aragon Nest Proposal:
Establishing a new Digital Body Corporate Legal Personality in the EU without Incorporation per #51 formerly #48

Abstract

By very nature a DAO is supposed to be a decentralized autonomous organization this definition is contrary to the definition of a corporation which has centralized control... even publicly traded companies with shareholders do not even scratch the surface of true decentralization.  The current environment finds most non-incorporated blockchains unable to utilize traditional financial or legal institutions or to enter litigation to defend itself if necessary.  Nearly finalized legislation in Malta an open-border EU member state will soon allow DAOs and other innovative technologies to achieve their own legal personality without incorporating.  This digital legal persona and its right to exist and conduct business should then be applicable for participation by residents of other EU member states as is the case with other cross border companies, foundations, and persons.

Deliverables

A DAO that is not incorporated and truly controlled by the community of token holders hosted on Aragon and registered with its own legal personality and right to exist and conduct business in the European Union. Our DAO is called TechCollect (formerly ETF Token)

Grant size

Funding: No salaries... we were doing this anyways :) Maybe help with some of the audit/agent verification costs and requirements of the proposed legislation, we decided to forego the ICO and deliver a working product issued by a self-funded DAO at fair value... so funds are tight.

Application requirements

Project specifics included in whitepaper:
https://www.etftoken.org/whitepaper

Development timeline

The development timeline will be the following one in regards to each deliverable:

1. July 15th - Aug 15th: Finish application and solicit quotes from registered agent and registered systems auditor. Create easy to follow step-by step instruction for achieving this digital body corporate legal status.
2. Aug 15th: Apply
3. Aug 31st: Receive Legal Personality
4. Sep 1st - 30th: Finish Development (deploy on Main net?)
5. Oct 1st - Nov 30th: Audit
6. Dec 1st: Apply for formal permission to make TECO Token available to retail consumers in the EU

**Comment by mariapao** [comment]
Hi @jtiannelli Thank you for submitting this proposal. 

You are right, the idea with proposal #48 is not to fund a project so they set up their legal entity. We want a well thought solution and PoC. If such solution works the idea is that any DAO, any project, will be able to use it. For this reason we are not going to move forward with this proposal.

**Comment by jtiannelli** [comment]
@mariapao Thanks for your time! And I’ll let you know when we’re done, like
I said we are very close to being registered and are still working on a
step-by-step guide, even if you guys are not particularly interested in
using it.   Maybe me posting a grant that didn’t include a request for
funding didn’t make sense in the first place? I was trying to find a way to
start sharing some of this information via the github style you guys use.

I would encourage a similar approach to be taken forward by your future
awardees where the main audit is performed of Aragon and then that auditor
offers a discount to other DAOs built on Aragon because the amount of code
they have to review is small.  Obviously we would be willing to share with
them how we got our EU blockchain registration  without using conventional
companies to further help in any way we can.  You can also share any of the
legal level breakdown documents we provided, as Malta is obviously the best
route for achieving this...  however I would encourage someone to do what
we are doing and start preparing friendlier looking documents in layman‘s
terms.

Keep soaring!
Best,
-JT
